tests/backend/test_supabase_integration.py

```
@@ -0,0 +1,620 @@
+
+    @pytest.mark.asyncio
+    async def test_api_messages_with_supabase(self):
+        """
+        SupabaseバックエンドでのAPI メッセージ履歴取得テスト
+
+        Test: Supabaseをバックエンドとしてメッセージ履歴APIが正常動作する
+        """
+        # Arrange
+        channel_id = "test_channel_1"  # noqa: F841
+        with pytest.raises(ImportError):
+                from src.backend.supabase_main import app
```

Ruff (0.11.9)

`src.backend.supabase_main.app` imported but unused

Remove unused import: `src.backend.supabase_main.app`

(F401)